.eslintrc.json

```
@@ -34,7 +33,15 @@
     "no-undef": "off",
     "@typescript-eslint/explicit-function-return-type": "off",
     "@typescript-eslint/explicit-module-boundary-types": "off",
-    "@typescript-eslint/no-use-before-define": "warn",
+    "@typescript-eslint/no-use-before-define": [
+      "warn",
+      {
+        "functions": false,
+        "classes": false,
+        "variables": false,
+        "typedefs": false
```

```suggestion
        "variables": false
```
Tested locally that eslint passes without these two.